client/app/states/dashboard/dashboard.state.js

```diff
@@ -90,34 +130,38 @@
     if (!$state.navFeatures.services.show) {
       return undefined;
     }
-    var options = {expand: false, filter: ['retired=true'] };
+    var options = {expand: false, filter: ['service_id=nil', 'retired=true'] };
 
     return CollectionsApi.query('services', options);
   }
 
   /** @ngInject */
-  function resolveNonRetiredServices(CollectionsApi, $state) {
+  function resolveServicesWithDefinedServiceIds(CollectionsApi, $state) {
     if (!$state.navFeatures.services.show) {
       return undefined;
     }
-    var options = {expand: false, filter: ['retired=false'] };
+    var options = {expand: false, filter: ['service_id=nil'] };
 
     return CollectionsApi.query('services', options);
   }
 
-  /** @ngInject */
-  function resolveServicesWithDefinedServiceIds(CollectionsApi, $state) {
-    if (!$state.navFeatures.services.show) {
-      return undefined;
+  function chainRequestPromises(promiseArray, vm, type) {
```

Comment:
Might prefer to use `$q.all` here, so that we don't hardcode the numer of promises in the array...

Something along the lines of

```js
$q.all(promiseArray).then(function(values) {
  var count = _.sumBy(values, 'subcount');
  vm.requestsCount[type] = count;
 vm.requestsCount.total += count;
});
```